src/context/json.test.ts

```diff
@@ -3,18 +3,28 @@ import { response } from '../response'
 
 describe('json', () => {
   describe('given a JSON body', () => {
-    let result: ReturnType<typeof response>
-
-    beforeAll(() => {
-      result = response(json({ firstName: 'John' }))
-    })
-
     it('should have "Content-Type" as "application/json"', () => {
+      const result = response(json({ firstName: 'John' }))
+
       expect(result.headers.get('content-type')).toEqual('application/json')
     })
 
     it('should have body set to the given JSON', () => {
-      expect(result).toHaveProperty('body', `{"firstName":"John"}`)
+      const object = response(json({ firstName: 'John' }))
+
+      expect(object).toHaveProperty('body', `{"firstName":"John"}`)
```

Comment:
I would truncate these assertions to call `response()` within the `expect`. It doesn't read right, as calling the `response` function always returns a response, not object/array/etc.

```js
expect(response(json({ firstName: 'John' }))).toHaveProperty('body', `{"firstName":"John"}`)
```